Sources/SpeziOnboarding/ConsentView/ConsentDocumentModel.swift

```diff
@@ -0,0 +1,29 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import TPPDF
+
+/// A type holding all information required for exporting a PDF from a `ConsentDocumentVIew`, such as the PDF content (i.e., markdown string) and the ExportConfiguration.
+public struct ConsentDocumentModel {
```

Comment:
Wondering if we need to make this module public; I could see that we can be fine with an internal struct & methods and do information hiding by making most of the functions private as you did above.